### PR TITLE
Sepa transfer for purchases

### PR DESCRIPTION
**Pull request by jonathanpa**
Some notes:
* A `t.column :sepa` in the `BackendOutgoingPaymentModesController list` raises an `undefined method `text' for "<div class=\"checkbox-true\"></div>":ActiveSupport::SafeBuffer`
* For the dropdown "Pay" button in the `OutgoingPaymentList` index view, I didn't way how to make
the button dropdownable with the existing helper. Consequently, I take the first `OutgoingPaymentMode` in SEPA.
  * Should I take the `active == true` `OutgoingPaymentMode` ? I see in fixtures they have all `active: false`.
  * I take the purchases whose `state` is `order`. When should we move the `state` to `invoice` ?
* For printing, I tried to follow what was done for the purchases or the inventories without success. By the way, to print purchase or inventory raises me an error on my machine.

### CODE REVIEW

File: app/controllers/backend/outgoing_payment_modes_controller.rb [hunk 1]
```diff
@@ -31,6 +31,7 @@ class OutgoingPaymentModesController < Backend::BaseController
       t.column :name
       t.column :cash, url: true
       t.column :with_accounting
+      # t.column :sepa
```
Comment (jonathanpa) [inline app/controllers/backend/outgoing_payment_modes_controller.rb]: If enable I get an `undefined method 'text' for "<div class=\"checkbox-true\"></div>":ActiveSupport::SafeBuffer`.
Is `t.column :sepa` the correct way to add a "boolean" column to the table ?

File: app/controllers/backend/outgoing_payment_modes_controller.rb [hunk 2]
```diff
@@ -31,6 +31,7 @@ class OutgoingPaymentModesController < Backend::BaseController
       t.column :name
       t.column :cash, url: true
       t.column :with_accounting
+      t.column :sepa
```
Comment (Aquaj) [inline app/controllers/backend/outgoing_payment_modes_controller.rb]: Appears to be working. (@ 56eebae)

File: app/controllers/backend/outgoing_payment_modes_controller.rb (hunk 1 above)
Comment (Aquaj) [inline app/controllers/backend/outgoing_payment_modes_controller.rb]: See previous review on same line. Fixed as of now (56eebae), probably by 3da1d44.

**Comment by Aquaj** [comment]
Reviewed. Would like another review (cc @igkyab ).

**Comment by igkyab** [comment]
Reviewed too. I merge the branch.

**Pull request merged**